projects/plugins/jetpack/extensions/blocks/sharing-button/components/sms-icon.js

```diff
@@ -0,0 +1,17 @@
+import { Path, SVG } from '@wordpress/components';
```

Comment:
What do you think about adding that icon to [the Social Icons package](https://github.com/Automattic/social-logos/)? We already have a feed and an email icon, I think an SMS icon could be a good addition.

Reply:
Yeah, I agree. I just wasn't sure about the process and if we should.

Reply:
I've added the new button to the package in https://github.com/Automattic/social-logos/pull/131 and created #36176 to bring in that new package version to Jetpack.

Reply:
Done in 012f29e0c40b12f1da2c3049e3ae4146f761bdb8